.test-infra/jenkins/CommonTestProperties.groovy

```diff
@@ -29,7 +29,7 @@ class CommonTestProperties {
         DATAFLOW("DataflowRunner"),
         SPARK("SparkRunner"),
         SPARK_STRUCTURED_STREAMING("SparkStructuredStreamingRunner"),
-        FLINK("TestFlinkRunner"),
+        FLINK("FlinkRunner"),
```

Comment:
This string broke the result reporting because it wrote to the wrong BigQuery table?

Reply:
Together with setting `--runner=TestFlinkRunner`, this should suffice to make it work again.

Reply:
Yes exactly this wrote to a different table than the historic one. Also having the non Test version will make it uniform with all the other runners + having the parallelism not limited to 1 which gave the impression of worse performance.